apps/authentication-portal/src/main/webapp/handle-multiple-sessions.jsp

```diff
@@ -204,6 +190,22 @@
     <% } %>
 
     <script>
+        $(document).ready(function() {
+            <c:forEach items='${requestScope.data["sessions"]}' var="session" varStatus="loop">
+                var accessedTime = getDateFromTimestamp(<e:forJavaScript value="${session[1]}"/>);
+                var tableRow =
+                    "<tr>"
+                    + "<td><e:forHtmlContent value="${loop.index + 1}"/></td>"
+                    + "<td><e:forHtmlContent value="${session[2]}"/></td>"
+                    + "<td><e:forHtmlContent value="${session[3]}"/></td>"
+                    + "<td id='<e:forHtmlAttribute value="${session[1]}"/>'>" + accessedTime + "</td>"
+                    + "<td><input type='checkbox' onchange='toggleMasterCheckbox()' value='<e:forHtmlAttribute value="${session[0]}"/>' name='sessionsToTerminate' checked/></td>"
+                    + "</tr>";
+
+                $("#session-details tbody").append(tableRow)
```

Comment:
Let’s add the semicolon here.

Reply:
Resolved by https://github.com/wso2/identity-apps/pull/3612/commits/17b1be24446142e8c7af2dc9faa3d4ad1392035f